src/app/containers/Ad/Amp/index.jsx

```diff
@@ -36,29 +63,54 @@ const ampAdPropsDesktop = ({ service }) => ({
   json: JSON.stringify(constructAdJsonData({ service })),
 });
 
+const AMP_ACCESS_DATA = (endpoint) => ({
+  authorization: endpoint,
+  noPingback: true,
+});
+
+const AMP_ACCESS_FETCH = (service) => {
+  const togglesEndpoint = `https://toggles.test.api.bbci.co.uk/toggles?application=simorgh&service=${service}&geoiplookup=true`;
+
+  return (
+    <script id="amp-access" type="application/json">
+      {JSON.stringify(AMP_ACCESS_DATA(togglesEndpoint))}
+    </script>
+  );
+};
+
 // eslint-disable-next-line react/prop-types
 const AmpAd = ({ service }) => {
   return (
     <>
-      <amp-ad {...ampAdPropsMobile({ service })}>
-        <amp-img
-          placeholder
-          width="320"
-          height="50"
-          src="https://via.placeholder.com/320x50"
-          layout="responsive"
-        />
-      </amp-ad>
-
-      <amp-ad {...ampAdPropsDesktop({ service })}>
-        <amp-img
-          placeholder
-          width="970"
-          height="250"
-          src="https://via.placeholder.com/970x250"
-          layout="responsive"
-        />
-      </amp-ad>
+      <Helmet>
+        {AMP_ACCESS_JS}
+        {AMP_ACCESS_FETCH(service)}
+      </Helmet>
+      <div
+        amp-access="toggles.ads.enabled AND geoIp.advertiseCombined"
+        amp-access-hide="true"
```

Comment:
I've added a string here because when I run the unit tests for Ads I get this console warning...
![image](https://user-images.githubusercontent.com/30599794/78269494-95640a80-7501-11ea-8888-4459a633823d.png)

Reply:
And this attribute hides the `div` by default, until we get a successful authorization response right?

Reply:
Yep that is correct 💪

Reply:
I've just gone down a bit of a rabbit hole looking into this - it looks like react is buggy around boolean, non-standard, html attributes (i.e. `amp-access-hide` isn't an [html attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes)). See here: https://github.com/facebook/react/issues/9230 - they recommend using `amp-access-hide=""` instead, which is rendered as `amp-access-hide`. This is a bit rubbish, not sure which I prefer, as `amp-access-hide="true"` isn't quite what we should be passing to the AMP component (and `amp-access-hide="false"` wouldn't have the opposite effect, due to how boolean attributes work)